tox.ini

```diff
@@ -0,0 +1,69 @@
+[tox]
+envlist =
+    py{39,310,311}
```

Comment:
We have 3.9 here but setup.cfg says 3.8, are you planning to drop 3.8 support?

Reply:
Well I didn't see any reason to exclude 3.8 but also didn't want to add an additional test. I guess I should just only support 3.9+ if I'm not testing it.